pkg/traceability/sampling/errors.go

```diff
@@ -5,5 +5,5 @@ import "github.com/Axway/agent-sdk/pkg/util/errors"
 // Config errors
 var (
 	ErrGlobalSamplingCfg = errors.New(1520, "the global sampling config has not been initialized")
-	ErrSamplingCfg       = errors.New(1521, "sampling percentage must be between 0 and 100")
+	ErrSamplingCfg       = errors.New(1521, "sampling percentage must be between 0 and 10.  Setting sampling percentage to default value of 1 percent")
```

Comment:
maybe this error string should be a format string using the values of the default and maximum constants?

Reply:
I can get with that